### PR TITLE
PHP: add php-cs-fixer config and apply php-cs-fixer

### PR DESCRIPTION
**Pull request by thinkerou**
Now we have had `run_php_cs_fixer.sh`, but some codes are not the result which run it.
For `tests/qps/generated_code` are not executed fixer, so add config file `.php_cs.dist`.
@ZhouyihaiDing please review the pull request, thanks!

### CODE REVIEW

File: src/php/.php_cs.dist [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+$finder = PhpCsFixer\Finder::create()
+    ->exclude('bin')
+    ->exclude('ext')
+    ->exclude('tests/data')
+    ->exclude('tests/qps/generated_code')
+    ->in(__DIR__)
+;
+
+return PhpCsFixer\Config::create()
```
Comment (ZhouyihaiDing) [inline src/php/.php_cs.dist]: Thanks for the pull request!
What do you think about changing the settings to [this config](https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/.php_cs.dist)?

Comment (thinkerou) [reply src/php/.php_cs.dist]: Using config file in order to ensure as far as possible not change existing code.
1. use `exclude` for not run php-fixer for generated php file by proto
2. use `setUsingCache` for re-run every time
3. use `@Symfony` and `phpdoc_no_package` and `yoda_style` for not change existing code.

**Comment by thinkerou** [comment]
1. If not use `'phpdoc_no_package' => false`, (here set false) it will change the code:

```php
--- a/src/php/lib/Grpc/AbstractCall.php
+++ b/src/php/lib/Grpc/AbstractCall.php
@@ -22,7 +22,6 @@ namespace Grpc;
 /**
  * Class AbstractCall.
  *
- * @package Grpc
  */
 abstract class AbstractCall
 {
```

2. If not use `'yoda_style' => false`, it will change the code:

```php
--- a/src/php/lib/Grpc/BidiStreamingCall.php
+++ b/src/php/lib/Grpc/BidiStreamingCall.php
@@ -46,11 +46,11 @@ class BidiStreamingCall extends AbstractCall
     public function read()
     {
         $batch = [OP_RECV_MESSAGE => true];
-        if ($this->metadata === null) {
+        if (null === $this->metadata) {
             $batch[OP_RECV_INITIAL_METADATA] = true;
         }
         $read_event = $this->call->startBatch($batch);
-        if ($this->metadata === null) {
+        if (null === $this->metadata) {
             $this->metadata = $read_event->metadata;
         }
```

3. Use `@Symfony` ,please see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3758

**Comment by ZhouyihaiDing** [comment]
I think we are planning to conform to `PSR2`.

[phpdoc_no_package](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/5172d8d19792b3d5b6f2207d45125a1afc75e25f/src/RuleSet.php#L130) and [yoda_style](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/5172d8d19792b3d5b6f2207d45125a1afc75e25f/src/RuleSet.php#L161) are enabled by `Sympony`. The issue you mentioned is strange, because `Symfony` enables `PSR2` as well, which should get the same result as default(`PSR2`), I guess there are some other rules enabled/disabled by [Sympony](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/5172d8d19792b3d5b6f2207d45125a1afc75e25f/src/RuleSet.php#L58) which fix the issue.

**Comment by thinkerou** [comment]
Yes, we can set Sympony rule, but I think the pr changed codes which we should need all.

**Comment by stanley-cheung** [comment]
Sorry I need to declare PR bankruptcy on some of these older PRs. Running php-cs-fixer is a good idea, but this is a bit too invasive. We will see whether this is a better, safer time to do this.

**Pull request closed without merging**